Components/Rating/rating.css

```diff
@@ -0,0 +1,16 @@
+i:hover {
+  color: var(--gold-color);
+}
+
+i {
```

Comment:
instead of selecting using 'i', you can use classes example-star, rating-star etc. to select the exact element.